[2024-04-26T04:15:22.641+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T04:15:22.675+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:15:19.453704+00:00 [queued]>
[2024-04-26T04:15:22.688+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:15:19.453704+00:00 [queued]>
[2024-04-26T04:15:22.690+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-26T04:15:22.709+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_data> on 2024-04-26 04:15:19.453704+00:00
[2024-04-26T04:15:22.717+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3712) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-26T04:15:22.718+0000] {standard_task_runner.py:63} INFO - Started process 3716 to run task
[2024-04-26T04:15:22.719+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Crypto_ETL', 'fetch_and_save_data', 'manual__2024-04-26T04:15:19.453704+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/processing_ETL.py', '--cfg-path', '/tmp/tmpf4fddnja']
[2024-04-26T04:15:22.720+0000] {standard_task_runner.py:91} INFO - Job 12: Subtask fetch_and_save_data
[2024-04-26T04:15:22.800+0000] {task_command.py:426} INFO - Running <TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:15:19.453704+00:00 [running]> on host e30f8265f0c9
[2024-04-26T04:15:22.925+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Crypto_ETL' AIRFLOW_CTX_TASK_ID='fetch_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T04:15:19.453704+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-26T04:15:19.453704+00:00'
[2024-04-26T04:15:22.926+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T04:15:22.994+0000] {logging_mixin.py:188} INFO - Data saved to binance_dataset.csv
[2024-04-26T04:15:22.995+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-26T04:15:22.995+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T04:15:23.005+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Crypto_ETL, task_id=fetch_and_save_data, execution_date=20240426T041519, start_date=20240426T041522, end_date=20240426T041523
[2024-04-26T04:15:23.056+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T04:15:23.077+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T04:15:23.081+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
